dbt/adapters/athena/column.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from dbt.adapters.athena.relation import TableType
+from dbt.adapters.base.column import Column
+from dbt.exceptions import DbtRuntimeError
+
+
+@dataclass
+class AthenaColumn(Column):
```

Comment:
Is there a risk to have discrepancies with the types defined in pyathena ?
Not sure it has a real impact but it could lead to confusion .

Reply:
@Jrmyy I've tested the changes on my production models (~100 models) and they just work fine